### ISSUE TITLE
Unable to pass payload config inside payload init function

### ISSUE THREAD

**Issue by anjy7** [payloadcms/payload-3.0-alpha-demo#31]
```
import { getPayload } from 'payload'

import payloadConfig from '../../payload.config.js'

export const getPayloadClient = () => getPayload({ config: payloadConfig })
```
Upon running the local api script outside the payload project, it throws scss errors:
![image](https://github.com/payloadcms/payload-3.0-alpha-demo/assets/92802904/6b531707-e653-49a6-b5e6-051286ff2ae6)

Steps to reproduce-
1. Create a local api script to seed the data 
2.  Run the script using ‘node script.js’ or ‘ts-node script.js’

**Comment by paulpopus** [comment]
What a head spinner this one was, so here's the current fix for this, you'll need to create a custom loadConfig function that tells node to ignore css files:

```ts
// loadConfig.ts

import { createRequire } from 'module'

export const loadConfig = async (path: string) => {
  const require = createRequire(import.meta.url)

  const CLIENT_EXTENSIONS = [
    '.scss',
    '.css',
    '.svg',
    '.png',
    '.jpg',
    '.eot',
    '.ttf',
    '.woff',
    '.woff2',
  ]

  CLIENT_EXTENSIONS.forEach((ext) => {
    require.extensions[ext] = () => null
  })

  const config = require(path).default

  return config
}
```

then in your script you wanna import the config using this function instead, eg:

```ts
// src/seed.ts
import { getPayload } from 'payload'
import dotenv from 'dotenv'
import { loadConfig } from './loadConfig'

dotenv.config()

export const seed = async () => {
  const config = await loadConfig('../payload.config')
  const payload = await getPayload({ config })

  const pages = await payload.find({
    collection: 'pages',
  })

  console.log(pages.docs)
  process.exit(1)
}

seed()
```

then you should be able to run the script with `tsx src/seed`


We are planning to streamline this whole process but currently this is the best way to do it

**Comment by torbjorn-kvist** [comment]
@paulpopus sorry for tagging, but wanted to ask if there is a "easy" way to follow or get notified about the streamlined process or is that something that will be "included" in the 3.0 release?

**Comment by torbjorn-kvist** [comment]
Okey! Thanks for the response!

I'm mostly asking because the "closed" answer is for a script running via `tsx`. 
I did some testing today with trying to use "local api" with v3 inside a firebase function. 
```typescript

import { defineSecret } from 'firebase-functions/params'
import { setGlobalOptions } from 'firebase-functions/v2'
import { onRequest } from 'firebase-functions/v2/https'
import { getPayload } from 'payload'

import config from '@/payload.config'

const MONGODB_URI = defineSecret('MONGODB_URI')
const PAYLOAD_SECRET = defineSecret('PAYLOAD_SECRET')

setGlobalOptions({
  region: 'europe-west1',
  secrets: [MONGODB_URI, PAYLOAD_SECRET],
  invoker: 'public',
})

export const testLocalApi = onRequest({ minInstances: 0 }, async (request, response) => {
  const payload = await getPayload({
    config: config,
  })

  const data = await payload.find({
    collection: 'users',
  })

  response.send(data)
})

```

Got the same error with `.scss` as the seed script mention in the issue.
Tried to the do the loadConfig solution but got an error, which makes sense that it works in tsx but not inside an firebase function. 
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /user/path/worker/src/payload.config.js from /user/path/worker/loadConfig.js not supported.
Instead change the require of payload.config.js in /user/path/worker/loadConfig.js to a dynamic import() which is available in all CommonJS modules.
``` 

Not sure if this it quite out-of-scope of what Payload is targeting.
